apps/web/src/global.d.ts

```diff
@@ -32,6 +32,7 @@ type DesktopIntegrationSettings = {
 };
 
 declare interface Window {
+  os: string | undefined;
```

Comment:
this `os` property is basically the same as `process.platform` in Node.JS so we should use the typings from that.

Reply:
Ok